x-pack/solutions/observability/plugins/streams_app/public/components/stream_detail_enrichment/page_content.tsx

```diff
@@ -200,16 +196,7 @@ const ProcessorsEditor = React.memo(
 
     return (
       <>
-        <EuiPanel
-          paddingSize="m"
-          hasShadow={false}
-          borderRadius="none"
-          grow={false}
-          css={css`
-            z-index: ${euiTheme.levels.maskBelowHeader};
-            ${useEuiShadow('xs')};
-          `}
-        >
+        <EuiPanel paddingSize="m" hasShadow={false} grow={false} color="transparent">
```

Comment:
The shadow on this header was meant to have a marked once we start scrolling a processor list that is longer than the available space, without this the processors entries would cross an invisible line that looks a bit weird.

<img width="450" alt="Screenshot 2025-02-19 at 17 29 23" src="https://github.com/user-attachments/assets/dc97d3d5-1395-4cd6-9bad-0a78d3e2e612" />